Source/Core/Xml.cs

```diff
@@ -126,16 +127,30 @@ public void WriteSplit(int splitNum, DateTime startTime, string outcome, TimeSpa
 
       cce.BeginExpose(this);
       {
-        wr.WriteStartElement("split");
+        wr.WriteStartElement("assertionBatch");
```

Comment:
Just for the record, this will be a breaking change for any consumers processing this XML, but I'm fine with that. It's such an easy change to account for, and I'm not aware of any consumers besides Dafny processing this output.

Reply:
Yeah, it's true. Dafny will need to change, but it'll be easy. I take it that you're happy with the terminology?